recipes/numactl-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: numactl-cos6-x86_64
+  version: 2.0.9
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/numactl-2.0.9-2.el6.x86_64.rpm
+    sha256: 70e58c3b01ce62d90ef0e3d9f1c0d1d3efaa4d237b3b821ee7d1e791553d82a7
+    folder: binary
+
+build:
+  number: 0
+  noarch: generic
+  missing_dso_whitelist:
+    - '*'
+
+test:
+  commands:
+    - test -d "${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot"
+  requires:
+    - zlib
+
+about:
+  home: ftp://oss.sgi.com/www/projects/libnuma/download
+  license: LGPLv2/GPLv2
+  license_family: GPL2
```

Comment:
@isuruf, I looked for a license file in source rpm and the GitHub repo and did not find one unfortunately. Can we skip this requirement here?

Reply:
You can copy the GPL text from and add it to the recipe file. Use the version as described in https://github.com/numactl/numactl#license-copyrights-acknowledgements

Reply:
That's not the license file though. It's just text that notes how it has been licensed.

FWIW I did raise an issue to ask them to add the license. ( https://github.com/numactl/numactl/issues/68 )

Reply:
I mean the README says GPL 2.0, so you can add the GPL 2.0 text from gnu.org and add it to recipe

Reply:
We shouldn't be creating a license file for another project this way. They may chose to make exceptions or other changes. We may get the copyright dates invalid. We may not know the full author list or what companies hold rights on the project. There may be other things we botch by trying to do this.

It's also worth noting `libnuma` is LGPL 2.1 and `numactl` is GPL 2.

Reply:
cc @scopatz (for awareness)

Reply:
> We shouldn't be creating a license file for another project this way

Agreed.  We absolutely cannot license a package for someone.  If they claim it is GPL, but don't provide the license, they are effectively breaking the license they claim to have.  In fact, claiming to be GPL but not having a license file would mean that the package is still under right of first publish.

Reply:
It is worth pointing out that we are not packing this ourselves. We are repackaging what CentOS 6 already packaged in their RPM. In fact there are [many distros that package `numactl`]( https://pkgs.org/download/numactl ). Somehow they have all justified doing this without the license file. How did they do it? What are we missing here? One thing that sticks out to me are [these lines]( https://github.com/numactl/numactl/blob/1959c774398fa6859ff98dde76c01c7579be2731/numa.h#L13-L15 ).

Reply:
> We shouldn't be creating a license file for another project this way

We are not. IANAL, so don't quote me on this. They are saying we should use GPL 2.0 license and we don't need to distribute the copyright notice with the conda package, but with GPL we are saying we'll provide the source which has the copyright notice if somebody asks.

We do this for many GPLed R packages. For eg: https://github.com/conda-forge/r-pvclust-feedstock/blob/4c0a4d0c30d02e8e5068cb0767bf752af12acd14/recipe/meta.yaml#L44
We don't do it for BSD licensed R packages, because the license text for BSD require the distribution of the copyright notice.

Reply:
Any thoughts on [my comment above]( https://github.com/conda-forge/staged-recipes/pull/8632#discussion_r300052236 )?

Reply:
Ok I haven't seen any traction on the upstream issue. Pinged it again just in case.

For now, here's what I have tried to do on the recipe end. I've included the copyright section that @isuruf pointed to. Also have included copies of the GPL 2 and LGPL 2.1 license files. (BTW thanks for getting this fixed in conda-build @isuruf! 😄)

The man pages also have license headers that include what is referred to as [the "verbatim" license]( https://www.kernel.org/doc/man-pages/licenses.html#verbatim ). I haven't packaged this additionally as the man pages already include it.

Also the copyright refers to `numademo` having special conditions and the header of the source file should be inspected. However looking at `numademo`, it appears [this is just under GPL2]( https://github.com/numactl/numactl/blob/e7bc3059e189709aba48fc5945215ec2a902adf2/numademo.c#L7-L10 ) like the rest of the `numactl` (non-`libnuma`) bits. So didn't do anything special for it either.

I hope that sufficiently addresses the license packaging with these CDTs. Please let me know if you have other questions.